Tools/CMake/AMReX_Defines.cmake

```diff
@@ -93,6 +93,9 @@ function ( set_amrex_defines )
       # Fortran/C mangling scheme
       #
       include( FortranCInterface )
+      if(NOT FortranCInterface_GLOBAL_FOUND)
+          message(FATAL_ERROR "Failed to find the Fortan C Interface -- check the CMakeError.log")
```

Comment:
Oh dang @JBlaschke Fort**r**an

Reply:
Eh @ax3l -- if they wanted people to spell the name right, they would have made it a single-letter

Reply:
as long as it's all caps I am all in.
F

Reply:
As long as J will be "Julia" instead of "Java", then count me in!